tasks/install_yum.yml

```diff
@@ -1,91 +1,92 @@
+    - name: PostgreSQL | Install all the required dependencies | yum
+      yum:
+        name: ["ca-certificates", "python-psycopg2", "python-pycurl", "glibc-common", "epel-release", "libselinux-python"]
+        state: present
 
-      - name: PostgreSQL | Add PostgreSQL repository | yum
-        yum_repository:
-           name: postgresql
-           description: postgresql yum repo
-           baseurl: "{{ postgresql_yum_repository_baseurl }}"
-           gpgkey: "{{ postgresql_yum_repository_gpgkey }}"
-        when: postgresql_install_repository
-        
-      - name: PostgreSQL | Make sure the dependencies are installed | yum
-        yum:
-         name: ["python-psycopg2", "python-pycurl", "glibc-common","libselinux-python"]
-         state: present
-         update_cache: yes
+    - name: PostgreSQL | Add PostgreSQL repository | yum
+      yum_repository:
+        name: postgresql
+        description: postgresql yum repo
+        baseurl: "{{ postgresql_yum_repository_baseurl }}"
+        gpgkey: "{{ postgresql_yum_repository_gpgkey }}"
+      when: postgresql_install_repository
 
-      - name: PostgreSQL | Install PostgreSQL | yum
-        yum:
-          name:
-            - "postgresql{{ postgresql_version_terse }}-server"
-            - "postgresql{{ postgresql_version_terse }}"
-            - "postgresql{{ postgresql_version_terse }}-contrib"
-          state: present
-        environment: "{{ postgresql_env }}"
+    - name: PostgreSQL | Make sure the dependencies are installed | yum
+      yum:
+        name: ["python-psycopg2", "python-pycurl", "glibc-common", "libselinux-python"]
+        state: present
+        update_cache: true
 
-      - name: PostgreSQL | Setup service users profile | yum
-        template:
-          src: pgsql_profile.j2
-          dest: "~{{ postgresql_service_user }}/.pgsql_profile"
-          owner: "{{ postgresql_service_user }}"
-          group: "{{ postgresql_service_group }}"
-          mode: 0700
-        when: postgresql_service_user_pgsql_profile
+    - name: PostgreSQL | Install PostgreSQL | yum
+      yum:
+        name:
+          - "postgresql{{ postgresql_version_terse }}-server"
+          - "postgresql{{ postgresql_version_terse }}"
+          - "postgresql{{ postgresql_version_terse }}-contrib"
+        state: present
+      environment: "{{ postgresql_env }}"
 
-      - name: PostgreSQL | Create ~/pgtab.example | yum
-        template:
-          src: pgtab.j2
-          dest: "~{{ postgresql_service_user }}/pgtab.example"
-          owner: "{{ postgresql_service_user }}"
-          group: "{{ postgresql_service_group }}"
-          mode: 0644
-        when: postgresql_service_user_pgsql_profile
+    - name: PostgreSQL | Setup service users profile | yum
+      template:
+        src: pgsql_profile.j2
+        dest: "~{{ postgresql_service_user }}/.pgsql_profile"
+        owner: "{{ postgresql_service_user }}"
+        group: "{{ postgresql_service_group }}"
+        mode: 0700
```

Comment:
What is a reason of 700 mode? In other places, 644 is used.

Reply:
This was an existing setting but this file is a bash script that set environment vars for the `postgres` user, so it needs to be executable.